app/routes/search.tsx

```diff
@@ -1,9 +1,44 @@
+import { LoaderFunction } from '@remix-run/node';
+import { useLoaderData } from '@remix-run/react';
+import { getEvents } from '~/models/events.server';
+
+export const loader: LoaderFunction = async ({ request }) => {
+  // Get the query string from the URL
+
+  const url = new URL(request.url);
+  const query = url.searchParams.get('q') as string;
+  console.log(query);
```

Comment:
You might want to delete `console.log` statements! We could also add this to our ESLint config.

Reply:
Usually use it for debugging. Sounds great!